values.yaml

```diff
@@ -358,10 +344,8 @@ solr:
     configsetName: ckanConfigSet
 
 
-
 postgresql:
-  # postgresql.enabled -- Flag to control whether to deploy PostgreSQL
-  enabled: true
```

Comment:
This sould be set to true on default so we can revert this.

Reply:
Yes, I know it looks like it's left over from the tests, I'll change it right away.